lib/ably/realtime/connection.rb

```diff
@@ -470,7 +470,7 @@ def create_websocket_transport
                 'false'
               end
 
-              url_params['clientId'] = client.auth.client_id if client.auth.has_client_id?
+              url_params.merge!(client.auth.client_id_header_sync) # RSA7e1
```

Comment:
The commit message describes this as a "refactor" but it's not, right? It's a behaviour change (bug fix).

Reply:
latest commit for the change https://github.com/ably/ably-ruby/pull/418/commits/1ff7d6b7877362714542a6fecd70654fa6136941

Reply:
I mean, that doesn't really change anything — the original commit is still there and still says "refactor", and this one doesn't clarify it. But I don't particularly want to go back and forth on this so will resolve — please could you be more careful with your usage of the word "refactor" in the future though?